modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeJSPFieldSetGroupCheck.testjsp

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
@TamyrisTorres, please remove this blank line

Reply:
resolved